tests/unit/docker/test_hub_build_level.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import docker
+import pytest
+import mock
+from mock import Mock, patch
+
+from jina.docker.hubio import HubIO
+from jina.enums import BuildTestLevel
+from jina.peapods import Pod
+from jina.executors import BaseExecutor
+from jina.executors import AnyExecutor
+from jina.parsers.hub import set_hub_build_parser
+
+@pytest.fixture
+def mock_load_config():
+    return BaseExecutor
+
+cli = docker.APIClient(base_url='unix://var/run/docker.sock')
+
+def test_hub_build_level_pass(monkeypatch, mock_load_config):
+
+    monkeypatch.setattr(BaseExecutor, "load_config", mock_load_config)
```

Comment:
What does this mock do?

Reply:
Removed the mock to use an actual `test-yaml` file for the `BaseExecutor` as discussed.